modules/azure/aks-global/aad-group.tf

```diff
@@ -1,35 +1,35 @@
 resource "azuread_group" "view" {
   for_each     = { for ns in var.namespaces : ns.name => ns }
-  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${each.value.name}${var.group_name_separator}view"
+  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${var.location_short}${var.group_name_separator}${each.value.name}${var.group_name_separator}view"
   #description             = "Members of this group will have view access to the ${each.value.name} namespace."
   prevent_duplicate_names = true
   security_enabled        = true
 }
 
 resource "azuread_group" "edit" {
   for_each     = { for ns in var.namespaces : ns.name => ns }
-  display_name = "${var.aks_group_name_prefix}${var.group_name_separator}${var.subscription_name}${var.group_name_separator}${var.environment}${var.group_name_separator}${each.value.name}${var.group_name_separator}edit"
```

Comment:
What happens in the AAD when this is changed? Is it just the name that gets updated or is it actually a add and removal?
I would assume that we have a number of customers that is dependent on these groups to provide access to there users.
If the name is just updated and the AAD group gid is the same I think this should not create any issues for them.
But if that is not the case we will need to search through the AAD and verify where the current groups are used and help the customers to migrate to the new group.

Reply:
I have refactored this part to not change the display name by making it global over regions in order to not bump into any potential problems. Besides, the AD groups does not need to have the regional dimension. Same group can be used in any region.